src/DataProtection/DataProtection.Autorest/custom/Cmdlets/Platform/BackupInstance/Update-AzDataProtectionBackupInstance.ps1

```diff
@@ -29,7 +29,15 @@ function Update-AzDataProtectionBackupInstance
 
         [Parameter(Mandatory=$false, HelpMessage='List of containers to be backed up inside the VaultStore. Use this parameter for DatasourceType AzureBlob.')]
         [System.String[]]
-        ${VaultedBackupContainer},        
+        ${VaultedBackupContainer},
+        
+        [Parameter(Mandatory=$false, HelpMessage='Resource guard operation request in the format similar to <ResourceGuard-ARMID>/dppModifyPolicy/default. Use this parameter when the operation is MUA protected.')]
+        [System.String[]]
+        ${ResourceGuardOperationRequest},
+
+        [Parameter(Mandatory=$false, HelpMessage='Parameter to authorize operations protected by cross tenant resource guard. Use command (Get-AzAccessToken -TenantId "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx").Token to fetch authorization token for different tenant.')]
+        [System.String]
+        ${Token},
```

Comment:
Token as plaintext string is prohibited cause it can be leaked by accident. Use securestring instead.
Note: you may also need to change the implementation to convert it back to normal string before sending the request.

```suggestion
        [Parameter(Mandatory=$false, HelpMessage='Parameter to authorize operations protected by cross tenant resource guard. Use command (Get-AzAccessToken -TenantId "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx" -AsSecureString).Token to fetch authorization token for different tenant.')]
        [System.Security.SecureString]
        ${Token},
```

Reply:
![image](https://github.com/user-attachments/assets/5d720012-4aa8-40f4-987d-aad82a15e4de)

we have other commands where we have used token parameter as a string. so if use this it will be a different experience for our users and if we change the existing ones it will be a breaking change. Please suggest what is best here

Reply:
Added the SecureToken parameter as discussed